app/src/main/java/org/wikipedia/readinglist/ReadingListFragment.java

```diff
@@ -522,14 +520,9 @@ public void onOffsetChanged(AppBarLayout appBarLayout, int verticalOffset) {
                 toolbarExpanded = false;
             }
 
-            recyclerView.post(() -> {
-                if (isAdded()) {
-                    DeviceUtil.updateStatusBarTheme(requireActivity(), toolbar, toolbarExpanded && transparentStatusBarEnabled);
-                    requireActivity().getWindow().clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
-                    requireActivity().getWindow().setStatusBarColor(transparentStatusBarEnabled
-                            ? Color.TRANSPARENT : ResourceUtil.getThemedColor(requireActivity(), R.attr.main_status_bar_color));
-                }
-            });
+            DeviceUtil.updateStatusBarTheme(requireActivity(), toolbar,
+                    (appBarLayout.getTotalScrollRange() + verticalOffset) > appBarLayout.getTotalScrollRange() / 2);
```

Comment:
Should we add `actionMode == null &&` into the logic since the list is still scrollable when multi-selecting items?

![Screenshot_1558380118](https://user-images.githubusercontent.com/2435576/58046363-ea6d7c80-7af9-11e9-8bc1-3a1da206e5de.png)

Reply:
Sure!